numpy/core/_add_newdocs.py

```diff
@@ -2943,7 +2943,7 @@
 
 add_newdoc('numpy.core.multiarray', 'ndarray', ('argmax',
     """
-    a.argmax(axis=None, out=None)
+    a.argmax(axis=None, out=None, keepdims=<no value>)
```

Comment:
This should be `False` as seen in the implementation [here](https://github.com/numpy/numpy/pull/19211).

```suggestion
    a.argmax(axis=None, out=None, keepdims=False)
```

Reply:
@HaoZeke are you sure? [np.argmax](https://numpy.org/devdocs/reference/generated/numpy.argmax.html?highlight=argmax#numpy.argmax) and [np.argmin](https://numpy.org/devdocs/reference/generated/numpy.argmin.html#numpy.argmin) both seem to use `keepdims=<no value>`.

Reply:
```suggestion
    a.argmax(axis=None, out=None, *, keepdims=False)
```
I see that all ndarray documentation has [keepdims=False](https://numpy.org/devdocs/reference/generated/numpy.ndarray.mean.html). To further keep consistency with current documentation we should specify that keepdims is a keyword only parameter such as in [np.argmax](https://numpy.org/devdocs/reference/generated/numpy.argmax.html?highlight=argmax#numpy.argmax) and [ndarray.mean](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.mean.html).

Reply:
> @HaoZeke are you sure? [np.argmax](https://numpy.org/devdocs/reference/generated/numpy.argmax.html?highlight=argmax#numpy.argmax) and [np.argmin](https://numpy.org/devdocs/reference/generated/numpy.argmin.html#numpy.argmin) both seem to use `keepdims=<no value>`.

We should check that against the implementations for the defaults, perhaps that could be an issue to open :)